Source/DafnyCore/Resolver/Resolver.cs

```diff
@@ -481,12 +481,19 @@ enum ValuetypeVariety {
       prog.CompileModules.Add(prog.BuiltIns.SystemModule);
       RevealAllInScope(prog.BuiltIns.SystemModule.TopLevelDecls, systemNameInfo.VisibilityScope);
       ResolveValuetypeDecls();
+
+      if (Options.Get(CommonOptionBag.TypeSystemRefresh)) {
```

Comment:
I didn't expect this change based on the PR details

Reply:
My bad. That wasn't intended until a future PR.